clientv3/clientv3util/util.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clientv3util
+
+import (
+	"github.com/coreos/etcd/clientv3"
+)
+
+// KeyExists returns whether a given key exists or not. It does this by checking
+// if the key `Version` is greater than 0. It is a useful guard in transactions.
+func KeyExists(key string) Cmp {
+	return clientv3.Compare(clientv3.Version(key), ">", 0)
+}
+
+// KeyNotExists returns whether a given key is missing or not.
+func KeyNotExists(key string) Cmp {
```

Comment:
`KeyMissing`? `Not`s in function names can be confusing since it's reasoning about a negation